src/index.js

```diff
@@ -171,23 +144,24 @@ class HCaptcha extends React.Component {
         scriptLocation
       } = this.props;
       const mountParams = {
+        render: 'explicit',
         apihost,
         assethost,
         endpoint,
         hl,
         host,
         imghost,
-        recaptchacompat: reCaptchaCompat === false? "off" : null,
+        recaptchacompat: reCaptchaCompat === false? 'off' : null,
         reportapi,
         sentry,
         custom,
         loadAsync,
         scriptLocation,
       };
 
-      mountCaptchaScript(mountParams)
-        .then(this.handleOnLoad)
-        .catch(this.handleError);
+      hCaptchaLoader(mountParams)
+        .then(this.handleOnLoad, this.handleError);
```

Comment:
Can we keep the `.catch` for clearer understanding?

Reply:
`.catch` won't handle error from `hCaptchaLoader` as we reject promise on error in loader package, while before we were returning a promise

Reply:
yes, there is already try/catch in `handleOnLoad` callback

Reply:
- added catch for the case of errors in handlers

- errors in `hCaptchaLoader` are being caught by reject handler (can be verified with tests)